packages/runner/cypress/integration/runner.ui.spec.js

```diff
@@ -405,43 +405,4 @@ describe('src/cypress/runner', () => {
       cy.get('.reporter').should('not.exist')
     })
   })
-
-  describe('inline spec list', () => {
```

Comment:
Why was this test removed?

Reply:
I don't think this feature or spec makes sense anymore. Basically, this was testing the ability to have inline spec list in the e2e runner. With how `specPattern`is implemented (similar to `--spec`), we only glob for the specs we want - in this case, when executing `runner.ui.spec.js` in run mode, that would be this spec file.

Previously, even when `--spec` was provided, we still globbed all the `testFiles`. Since we only glob for the relevant spec files now, in run mode, the inline spec list only shows 1 spec file, meaning you cannot switch between specs using the UI in run mode (which doesn't make much sense anyway).

Here's an image. We will not show the spec list during run mode (and previously we shouldn't have shown in anyway) so this test didn't make a ton of sense in the first place, and even less sense now.

<img width="1234" alt="image" src="https://user-images.githubusercontent.com/19196536/146096745-9fe3732c-fe8a-41de-931b-d76d819dbd00.png">

Reply:
I made sure to remove the code relevant to this `CypressInternal_UseInlineSpecList` too: https://github.com/cypress-io/cypress/pull/19319/commits/f326e88500fce17fe84dbf40be4f67cb5d220a81